docs/timeseries/initializing.rst

```diff
@@ -0,0 +1,277 @@
+.. _timeseries-initializing:
+
+Creating time series
+***************************************
+
+.. |Time| replace:: :class:`~astropy.time.Time`
+.. |TimeDelta| replace:: :class:`~astropy.time.TimeDelta`
+.. |Table| replace:: :class:`~astropy.table.Table`
+.. |QTable| replace:: :class:`~astropy.table.Table`
+.. |TimeSeries| replace:: :class:`~astropy.timeseries.TimeSeries`
+.. |BinnedTimeSeries| replace:: :class:`~astropy.timeseries.BinnedTimeSeries`
+
+Initializing a simple time series
+=================================
+
+The first type of time series that we will look at here is |TimeSeries|,
+which can be used for a time series which samples a continuous variable at
+discrete and instantaneous times. Initializing a |TimeSeries| can be done
+in the same ways as initializing a |Table| object (see :ref:`Data Tables <astropy-table>`),
+but additional arguments related to the times should be specified.
+
+Evenly sampled time series
+--------------------------
+
+The easiest way to construct an evenly sampled time series is to specify the
+start time, the time interval, and the number of samples, for evenly sampled
+time series::
+
+    >>> from astropy import units as u
+    >>> from astropy.timeseries import TimeSeries
+    >>> ts1 = TimeSeries(time='2016-03-22T12:30:31',
```

Comment:
I find this a bit confusing because `time` is sort of "overloaded" as terminology.  What about `time0` in this usage?  That makes it clear that it's the *start* time, as opposed to the middle or something like that.

Reply:
In my radio data reading classes, I've ended up with `start_time` and `stop_time` (in analogy with `slice.start` and `slice.stop`.

Reply:
I've added ``start_time`` to ``TimeSeries``. I'm going for ``time_*`` for all time-related attributes for consistency (good for any kind of alphabetical sorting)